ibaqpy/commands/correct_batches.py

```
@@ -0,0 +1,222 @@
+import re
+from email.policy import default
```

Ruff (0.8.2)

`email.policy.default` imported but unused

Remove unused import: `email.policy.default`

(F401)